src/Common/src/Interop/Unix/System.Security.Cryptography.Native/Interop.OpenSslVersion.cs

```diff
@@ -23,7 +23,7 @@ internal static Version OpenSslVersion
                     const string OpenSSL = "OpenSSL ";
 
                     // Skip OpenSSL part, and get the version string of format x.y.z
-                    if (!Version.TryParse(OpenSslVersionDescription().AsReadOnlySpan().Slice(OpenSSL.Length, 5), out s_opensslVersion))
+                    if (!Version.TryParse(OpenSslVersionDescription().AsSpan().Slice(OpenSSL.Length, 5), out s_opensslVersion))
```

Comment:
This should use the new overload of AsSpan that takes two ints. I know that for now you want to just do the renames, but I assume you will then scrub all ".AsSpan().Slice(" occurrences, correct?

Reply:
Opened https://github.com/dotnet/corefx/issues/27330 to track that.